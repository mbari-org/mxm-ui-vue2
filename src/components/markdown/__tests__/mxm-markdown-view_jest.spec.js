import { mount } from '@vue/test-utils'
import MxmMarkdownView from '../mxm-markdown-view'

describe('MxmMarkdownView', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(MxmMarkdownView)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('renders correctly', () => {
    const wrapper = mount(MxmMarkdownView)
    expect(wrapper.element).toMatchSnapshot()
  })
})
