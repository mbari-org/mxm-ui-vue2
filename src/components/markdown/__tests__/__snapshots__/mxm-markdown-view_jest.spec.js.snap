// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MxmMarkdownView renders correctly 1`] = `
<div>
  <div
    class="row no-wrap items-start"
    style="width: 100%;"
  >
    <div
      style="vertical-align: top;"
    >
      <div
        class="rounded-borders bg-light-blue-1"
      >
        <div
          class="markdownText q-pa-sm"
          style="min-width: 12em; min-height: 4em; white-space: normal; color: gray; font-style: italic; font-size: smaller;"
        >
          (No description)
        </div>
      </div>
    </div>
     
    <!---->
  </div>
   
</div>
`;
